polls/tool/src/java/org/sakaiproject/poll/tool/producers/PollOptionBatchProducer.java

```diff
@@ -0,0 +1,156 @@
+/**********************************************************************************
+Copyright (c) ${license.git.copyrightYears} ${holder}
+
+Licensed under the Educational Community License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+            http://opensource.org/licenses/ecl2
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ **********************************************************************************/
+
+package org.sakaiproject.poll.tool.producers;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+import org.sakaiproject.poll.logic.PollListManager;
+import org.sakaiproject.poll.model.Option;
+import org.sakaiproject.poll.model.Poll;
+import org.sakaiproject.poll.tool.params.OptionBatchViewParameters;
+import org.sakaiproject.poll.tool.params.PollViewParameters;
+import org.sakaiproject.poll.tool.params.VoteBean;
+
+import uk.org.ponder.localeutil.LocaleGetter;
+import uk.org.ponder.messageutil.MessageLocator;
+import uk.org.ponder.messageutil.TargettedMessageList;
+import uk.org.ponder.rsf.components.UIBranchContainer;
+import uk.org.ponder.rsf.components.UICommand;
+import uk.org.ponder.rsf.components.UIContainer;
+import uk.org.ponder.rsf.components.UIELBinding;
+import uk.org.ponder.rsf.components.UIForm;
+import uk.org.ponder.rsf.components.UIMessage;
+import uk.org.ponder.rsf.components.UIOutput;
+import uk.org.ponder.rsf.components.decorators.UIFreeAttributeDecorator;
+import uk.org.ponder.rsf.flow.ARIResult;
+import uk.org.ponder.rsf.flow.ActionResultInterceptor;
+import uk.org.ponder.rsf.flow.jsfnav.NavigationCase;
+import uk.org.ponder.rsf.flow.jsfnav.NavigationCaseReporter;
+import uk.org.ponder.rsf.view.ComponentChecker;
+import uk.org.ponder.rsf.view.ViewComponentProducer;
+import uk.org.ponder.rsf.viewstate.ViewParameters;
+import uk.org.ponder.rsf.viewstate.ViewParamsReporter;
+
+@Slf4j
+public class PollOptionBatchProducer implements ViewComponentProducer,ViewParamsReporter, NavigationCaseReporter, ActionResultInterceptor {
+
+    public static final String VIEW_ID = "pollOptionBatch";
+
+    @Setter
```

Comment:
You could just stick the Setter on the class

Reply:
What do you mean with this?

Reply:
"You can also put a @Getter and/or @Setter annotation on a class. In that case, it's as if you annotate all the non-static fields in that class with the annotation."

https://projectlombok.org/features/GetterSetter

Reply:
Sounds good thanks! I've implemented the Setter on a class level.

Reply:
Tested and works fine, thanks again.